src/app/games/epaminondas/tests/EpaminondasPositionalMinimax.spec.ts

```diff
@@ -47,4 +48,15 @@ describe('EpaminondasPositionalMinimax', () => {
         expect(bestMove).toEqual(expectedMove);
     });
 
+    SlowTest.it('should be able play against itself', () => {
+        minimaxTest({
+            rules: EpaminondasRules.get(),
+            minimax,
+            options: minimaxOptions,
+            config: defaultConfig,
+            turns: 200,
+            shouldFinish: true,
```

Comment:
![image](https://github.com/EveryBoard/EveryBoard/assets/41200522/5efb22a9-976d-4fb3-b78d-7d84ba45236e)
In der daad, très déséquilibré celui ci